shell/shared/grain.js

```diff
@@ -1314,8 +1314,12 @@ if (Meteor.isClient) {
             // Store apitoken id1 and template in session storage in the offer
             // template namespace under key id2.
             var key = "offerTemplate" + id2;
+            var grain = getActiveGrain(globalGrains.get());
+            var grainTitle = grain.title();
+            var slug = grainTitle.toLowerCase().trim().replace(/[^\w ]+/g,'').replace(/ +/g,'-');
```

Comment:
I think I'd prefer to instead just add `-` to the list of characters that don't get stripped - it'd be weird to convert `'` or `"` into `-`, especially if it's like `Drew's grain` -> `Drew-s-grain`.  

Maybe collapse any instances of `/-+/` into a single `-` as a followup?